public/scripts/controllers/homeController.js

```diff
@@ -1,10 +1,11 @@
 (function (module) {
-const homeController ={};
+  const homeController ={};
 
-homeController.initHomeIndex = function () {
-  Chicken.fetchAll();
-  $('.tab-content').hide();
-  $('#chickens').fadeIn();
-};
-module.homeController = homeController;
+  homeController.initHomeIndex = function () {
+    $('.filters').siblings().remove();
+    Chicken.checkETag();
+    $('.tab-content').hide();
```

Comment:
Nice handling of clearing the prior elements out, then checking to see what to load.